_includes/tutorials/aggregating-count/flinksql/markup/dev/create-movie-sales-by-title.adoc

```diff
@@ -0,0 +1,6 @@
+Since the output of our transient query looks right, the next step is to make the query persistent. This looks exactly like the transient query, except we first create a new table with the https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/connectors/table/upsert-kafka/[Upsert Kafka] connector and then `INSERT INTO` the table. We use the Upsert Kafka connector because we only care about the most recent aggregates for a given release year (the key column). The `INSERT INTO` statement returns to the CLI prompt right away, having created a persistent stream processing program running in the Flink cluster, continuously processing input records and updating the resulting `movie_ticket_sales_by_title` table.
```

Comment:
missed this before -- "because we only care about the most recent aggregate for a given title (the key column)"

Reply:
Good catch!